index.bs

```diff
@@ -736,10 +712,13 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
      1. Let |transport| be [=this=].
      1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`,
         then abort these steps.
-     1. Set |transport|'s [=[[State]]=] to `"closed"`.
      1. Let |session| be |transport|'s [=[[Session]]=].
      1. [=session/Terminate=] |session| with |closeInfo|.{{WebTransportCloseInfo/errorCode}}.
```

Comment:
Btw, does this sentence need to start with, "In parallel, "? I doubt this should block the main thread.

```suggestion
     1. In parallel, [=session/terminate=] |session| with |closeInfo|.{{WebTransportCloseInfo/errorCode}}.
```

Reply:
I don't have a strong opinion here. Accepting the suggestion.